src/common/GraphicalInformationSet.cpp

```diff
@@ -132,104 +165,165 @@ bool GraphicalInformationSet::hasGraphicalInformation(const common::AbstractObje
 	return false;
 }
 
-bool GraphicalInformationSet::hasDefaultColor(common::AbstractObject* targetObject) const
+bool GraphicalInformationSet::hasGraphicalInformation(const AbstractObject const* targetObject) const
```

Comment:
the const pointer is unuseful since is is a copy.

Reply:
Done